nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -542,55 +550,20 @@ Been a mess since 2018, we'll fix it someday (probably)
 	forceMove(get_turf(locate(250, y, z)))
 	//check_overmap_elegibility(TRUE)
 
-/obj/structure/overmap/fighter/proc/throw_pilot() //Used when yeeting a pilot out of an exploding ship
+/obj/structure/overmap/fighter/proc/throw_pilot(m_damage = 200) //Used when yeeting a pilot out of an exploding ship
```

Comment:
Just calling this var `damage` makes way more sense

Reply:
I called it that so it wouldn't conflict with anything in the future as damage is a very simple name.
stands for mob damage

Reply:
this is PEDANTIC which is supposed to be MY thing but I'll concede.. this time 🔪